Tests/PointFreeTests/BlogTests.swift

```diff
@@ -23,7 +23,7 @@ class BlogTests: TestCase {
   }
 
   func testBlogIndex() {
-    let conn = connection(from: request(to: .blog(.index), basicAuth: true))
```

Comment:
we used basic auth when first first building the blog, but it's no longer necessary.